### PR TITLE
Testing/align spacing and style

### PR DESCRIPTION
**Pull request by chaburkland**
This is a logic-free change, that only exists to beautify all static-frame tests by aligning their styles and spacing.

Now, all tests will be seperated by a single newline, and all `# ------ ... ----` comments will be padded by one newline.
Additionally, all tests will have a:

```python
if __name__ == '__main__':
    import unittest
    unittest.main()
```

entrypoint at the end of the file.

### CODE REVIEW

File: static_frame/profile/__main__.py [hunk 1]
```diff
@@ -1142,8 +1142,9 @@ def get_arg_parser() -> argparse.ArgumentParser:
 python3 test_performance.py SeriesIntFloat_dropna --profile
             '''
             )
+    choices = sorted(str(x).replace("<class '__main__.","").replace("'>", "") for x in Perf.__subclasses__())
     p.add_argument('patterns',
-            help='Names of classes to match using fn_match syntax',
+            help=f'Names of classes to match using fn_match syntax ({choices})',
```
[the remark this answers is not in the document]
Comment (flexatone) [reply static_frame/profile/__main__.py:1147]: I like this, but will not `argparse` complain if we try to use partial pattern like `Pivot*`?

[the remark this answers is not in the document]
Comment (chaburkland) [reply static_frame/profile/__main__.py:1147]: Nope! This is just a help message

[the remark this answers is not in the document]
Comment (flexatone) [reply static_frame/profile/__main__.py:1147]: I see, it is not actually setting the choices now. Makes sense!

**Comment by flexatone** [comment]
@chaburkland: Many thanks for this cleanup!

I see mypy is not happy....

**Comment by chaburkland** [comment]
> @chaburkland: Many thanks for this cleanup!
> 
> I see mypy is not happy....

I think this is cause default is also not happy. I'll handle these quality failure here

**Comment by flexatone** [comment]
@chaburkland : thanks, I did not realize those issues were on default... I have been working on my branch and assumed those failures were there. Thanks for these improvements!

**Pull request merged**